releases/release-1.14/release_team.md

```diff
@@ -0,0 +1,13 @@
+| **Role** | **Name** (**GitHub / Slack ID**)  | **Shadow Name(s) (GitHub / Slack ID)** |
+| ------ | ------ | ------ |
+| Lead | Aaron Crickenberger ([@spiffxp](https://github.com/spiffxp)) |  |
```

Comment:
As discussed on the most recent SIG Release meeting, I would really like to have a shadow before I am comfortable saying yes to this.

Reply:
@spiffxp -- [@BenTheElder has volunteered](https://github.com/kubernetes/sig-release/issues/372#issuecomment-440512857); just waiting for you to accept.

Reply:
ok, I accept so this can move forward, but am interested in another (non-Google) volunteer

Reply:
Thanks Aaron!